frontend/src/metabase/admin/settings/components/SettingsSingleSignOnForm.jsx

```diff
@@ -142,7 +142,7 @@ export default class SettingsSingleSignOnForm extends Component {
           />
           <div className="py3">
             <div className="flex align-center">
-              <p className="text-medium">{t`Allow users to sign up on their own if their Google account email address is from:`}</p>
+              <p className="text-medium">{t`Allow users to sign up on their own if their Google account email address is from (use commas to separate multiple domains):`}</p>
```

Comment:
Maybe we can rewrite this a little and add some punctuation instead of brackets.

```suggestion
              <p className="text-medium">{t`Allow users to sign up on their own, if their Google account email machtes the following domain(s). You may use commas to separate multiple domains:`}</p>
```